src/pkg/jobmonitor/model.go

```diff
@@ -0,0 +1,46 @@
+//  Copyright Project Harbor Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package jobmonitor
+
+// WorkerPool job service worker pool
+type WorkerPool struct {
+	PoolID      string `json:"pool_id"`
```

Comment:
curious about `PoolID` and `PID`.

Reply:
PID is the process ID, as described in swagger.yaml